package.json

```diff
@@ -13,7 +13,8 @@
   "scripts": {
     "start": "node ./bin/www",
     "startDev": "export DEBUG=biglotteryfund:*,express-session,connect:session-sequelize; nodemon --inspect ./bin/www",
-    "test": "npm audit && eslint . && jest --env=node --maxWorkers=4",
+    "test-unit": "export DB_CONNECTION_URI=sqlite://:memory && jest --env=node --maxWorkers=4",
```

Comment:
The mail service relies on the main server file in order to render html templates. The upshot of this is that if required in unit tests it tries to connect to a database, setting this is the only way of making the tests run reliably. Not a bad thing to set but feels…awkward. Not sure we can get around this unless we rendered the emails using nunjucks directly but this would prevent us from using filters.